src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,30 @@
+package calculator;
+
+
+import java.util.Scanner;
+
+public class Calculator {
+    public int calculate(String value){
+        String[] values=value.split(" ");
+        int result=Integer.valueOf(values[0]);
+
+        for(int i=1;i<values.length;i++){
+            if(i%2==0){
+                int number = Integer.parseInt(values[i]);
+                if(values[i-1].equals("+")) result +=number;
```

Comment:
위와 마찬가지로 원하는 기호가 들어오지 않는 경우도 생각해야 할 것 같아요!

Reply:
그리고 블럭 처리 생략은 나중에 문제가 생길 확률이 큽니다!
```suggestion
                if(values[i-1].equals("+")) {
                  result +=number;
                }
```